### ISSUE TITLE
Add an option to generate multiple image files with descriptive names

### ISSUE THREAD

**Issue by kateler** [stamen/maperture#120]
This is following up on the third suggestion in #119 (for Maperture to include a small set of recommended locations for showing off a map style). 

While designing map styles, it would save a lot of time if it were quick to gather a set of screenshots that give a good representation of how the styles look at various zoom levels and locations. Once the z/lat/long is determined for these recommended locations (in #119), I'd love to be able to: 
- manually open a map pane for each stylesheet I want to record (or more simply, just one map pane with a stylesheet I want to record)
- press a button to have a .zip file created that includes a large, high-resolution image of each location, with a descriptive filename that includes the date, stylesheet name, and z/lat/long. 
- edit a configuration file to change/add the set of z/lat/long values if I prefer to use different locations than the default 

This would make it faster to create screenshots for shareholder documentation. People could also save such a .zip file for each release to document changes.

**Comment by aparlato** [comment]
This is an interesting idea! Before we start brainstorming here, I just want to confirm that current functionality does not meet the needs here: Right now, you can set the same map to a series of different locations in separate panels, then screenshot all of them at once in a single PNG without the UI.

This sounds like you would instead want each image to be separate, higher resolution, and named? If this is the primary difference, I'm curious how high of a resolution would be needed/desirable for it to be a substantial improvement since current methods are dependent upon screen/window size.

**Comment by kateler** [comment]
Right, it would be nice if I could get the set of images at a resolution higher than I could get if they were all open in different panels (and therefore each smaller than my entire screen). 

Part of the request is getting higher resolution files, and the other is the efficiency improvement of being able to basically hit a "screenshot this style" button instead of having to manually open a set of locations one at a time and save each screenshot. Anytime there's a new version of a stylesheet, cartographers need to document and share it. Right now this is done in a bespoke way each time, using different locations/zooms/extents/resolutions.

**Comment by almccon** [comment]
Yeah, we discussed this at the Carto Squad brainstorm about Documentation https://www.figma.com/file/IZJh3sQVB2fCeOWwBeYsbB/Carto-squad-documentation-chat?node-id=0%3A1&t=kaQZYNAtVIuMWA8h-3

We also discussed possibly targeting a sample dataset (https://github.com/stamen/carto-tools/issues/7) for the screenshots.

**Comment by mulloverit** [comment]
**Initial convo**
* large lift, unclear payoff
* not something you need until you really need it
* we do use screenshots a lot, it would be good to have them be standardized and high res
* could implement screenshots on commit, but this may bloat repos/prs unnecessarily


**Potential use cases:**
* we screen shot a lot when we file PRs, which we do a lot
* producing visual change logs for client delivery


Potentially related to https://github.com/stamen/carto-tools/issues/64 -- @aparlato tapping you to pull this into Epic discussion
